docs-2.0/nebula-cloud/2.how-to-create-subsciption.md

```diff
@@ -0,0 +1,31 @@
+# How to create a subscription
```

Comment:
We usually use "Create a subscription". Use questions as titles for "What is xxx" topics and FAQs.